atriaapp/atriacalendar/models.py

```diff
@@ -119,8 +129,62 @@ def __str__(self):
         return self.title + ", " + self.location
 
 
-# Base class for organizations that use the Atria platform
-class AtriaOrganization(models.Model):
-    org_name = models.CharField(max_length=40)
-    date_joined = models.DateTimeField(default=timezone.now)
+# general announcements from an Org (will show up on a user's feed if they have an org relationship)
+class AtriaOrgAnnouncement(models.Model):
+    org_id = models.ForeignKey(AtriaOrganization, on_delete=models.CASCADE)
```

Comment:
I prefer to leave the `_id` off the name of `ForeignKey` fields. The underlying database column will be called `org_id` automatically for a foreign key, but the Python traversal returns an `AtriaOrganization` object, not an int, so this can be misleading.

Reply:
Removed _id